website/docs/components/copy/snippet/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,67 @@
+## Usage
+
+### When to use
+
+- Use the `CopySnippet` component when you want users to be able to copy code snippets or text, such as URLs, tokens, or IDs.
+- Use when users need to extract and use specific information like API keys, configuration settings, or unique identifiers.
+
+### When not to use
+
+- Avoid using a `CopySnippet` component to copy content outside its container.
+- Don't use a `CopySnippet` in a code block. Instead, use a [Copy Button](/components/copy/button).
+
+### Effective Usage
+
+- There's no limit to the number of `CopySnippet` components on a page, but maintain a balance to avoid overwhelming the user or cluttering the interface.
```

Comment:
```suggestion
- There’s no limit to the number of `CopySnippet` components on a page, but maintain a balance to avoid overwhelming the user or cluttering the interface.
```